examples/flask/playback_runner.py

```diff
@@ -0,0 +1,53 @@
+from examples.flask.web_services import tape_recorder
+from playback.studio.equalizer import ComparatorResult, EqualityStatus
+from playback.studio.equalizer_tuning import EqualizerTuner, EqualizerTuning
+from playback.studio.studio import PlaybackStudio
+from playback.tape_recorder import TapeRecorder
+
+
+class ContentBasedEqualizerTune(EqualizerTuner):
+
+    @staticmethod
+    def playback_function(recording):
+        """
+        Rerun the operation given the recording
+        """
+        # The class it self is saved as metadata, we instantiate it and trigger the post method
+        operation_class = recording.get_metadata()[TapeRecorder.OPERATION_CLASS]()
+        operation_class.post()
+
+    @staticmethod
+    def result_extractor(outputs):
+        """
```

Comment:
same comment, with a bit of work this can also be extracted away to PlaybackResult.get(output_alias)

Reply:
This is not always the case (look at job executor result extractor plugin)
But this can be a common utility. Contributions are welcome :)